09_object_scope.rb

```diff
@@ -14,10 +14,13 @@ def name
 
   def greet(other_person)
     #This local variable shadows the method 'name'
-    name = "Not my name"
+    name = @name
```

Comment:
well the point here was to shadow the method `name` with some random data - just imagine you could *not* reassign `name`